homeassistant/components/reolink/host.py

```diff
@@ -432,6 +432,15 @@ def unregister_webhook(self):
         webhook.async_unregister(self._hass, self.webhook_id)
         self.webhook_id = None
 
+    @property
+    def event_connection(self) -> str:
+        """Return the event connection type."""
+        if self._webhook_reachable:
+            return "ONVIF push"
```

Comment:
If you return "keys" instead and use the `ENUM` device_class with set options these can be translated

Reply:
done